scraper.rb

```diff
@@ -19,8 +19,9 @@ def scrape(h)
 end
 
 def scrape_list(term, url)
-  scrape(url => MembersPage).member_urls.each do |mem_url|
-    data = scrape(mem_url => MemberPage).to_h.merge(term: term)
+  scrape(url => MembersPage).seat_rows.each do |seat_row|
+    next if primary_member_url.empty?
```

Comment:
I've opened a separate pull request for this: https://github.com/everypolitician-scrapers/croatia-parliament/pull/25